tests/api/test_delete_apis.py

```diff
@@ -0,0 +1,91 @@
+import unittest
+import json
+
+from tests.setup_database import Setup
+from tests.utils import OpenEventTestCase
+from tests.auth_helper import login, register
+from tests.api.utils import create_event, get_path
+from tests.api.utils_post_data import *
+
+from open_event import current_app as app
+
+
+class TestDeleteApi(OpenEventTestCase):
+    """
+    Test Delete APIs for 200 (successful) status codes
+    """
+    def setUp(self):
+        self.app = Setup.create_app()
+        with app.test_request_context():
+            create_event()
+
+    def _login_user(self):
+        """
+        Register a user and login
+        """
+        register(self.app, 'test@example.com', 'test')
+        login(self.app, 'test@email.com', 'test')
```

Comment:
Yup, but we do need to create a fine-grained register method that just creates the account but not logs in. They don't affect any tests currently but just to be in clear.